src/groupByKey.ts

```diff
@@ -2,6 +2,13 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: string): GroupsMap<object> {
+  return items.reduce((hash, obj) => {
```

Comment:
Wow, I had no idea how to use reduce in this task! Cool

Reply:
why not forEach?

Reply:
just came to my head.  I will make with forEach)